Makefile

```diff
@@ -108,3 +112,7 @@ create-tmp-env-ci:
 # 16 is exit code for critical https://classic.yarnpkg.com/lang/en/docs/cli/audit
 audit:
 	bash -c 'yarn audit --level critical; [[ $$? -ge 16 ]] && exit 1 || exit 0'
+
+mine:
+	docker exec -it galoy-bitcoind-1 /bin/sh -c 'ADDR=$$(bitcoin-cli getnewaddress "") && bitcoin-cli generatetoaddress 6 $$ADDR '
```

Comment:
this does not work if you clone the repo in a different folder name. if we want to keep this in make file then you have to do something like 
```
container_id=$(docker ps -q -f status=running -f name="bitcoind")
docker exec $container_id bitcoin-cli listwallets
```

Reply:
fixed